android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java

```diff
@@ -403,7 +403,7 @@ public void openPicker(final ReadableMap options, final Promise promise) {
         setConfiguration(options);
         resultCollector.setup(promise, multiple);
 
-        permissionsCheck(activity, promise, Collections.singletonList(Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU ? Manifest.permission.WRITE_EXTERNAL_STORAGE : Manifest.permission.READ_MEDIA_IMAGES), new Callable<Void>() {
+        permissionsCheck(activity, promise, Collections.singletonList(Manifest.permission.WRITE_EXTERNAL_STORAGE), new Callable<Void>() {
```

Comment:
Shouldn't we open the picker directly in case is Android 13 + ?
```suggestion

       if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
           initiatePicker(activity);
       }

        permissionsCheck(activity, promise, Collections.singletonList(Manifest.permission.WRITE_EXTERNAL_STORAGE), new Callable<Void>() {
```